package.json

```diff
@@ -8,7 +8,7 @@
     "tailwindcss": "^4.0.5"
   },
   "scripts": {
-    "postinstall": "rsync -a --delete node_modules/* static/vendor/",
```

Comment:
This will not work on our prod machine or other development machines:

```
sh: 1: powershell: not found
npm error code 127
npm error path /home/sheena/workspace/pyconza2025-yokwejuste
npm error command failed
npm error command sh -c powershell -Command "if ($env:MSYSTEM -eq 'MINGW64') { rsync-win -a --delete node_modules/* static/vendor/ } else { rsync -a --delete node_modules/* static/vendor/ }"

npm error A complete log of this run can be found in: /home/sheena/.npm/_logs/2025-05-12T05_23_50_941Z-debug-0.log
```

The server is running the following:

```
Distributor ID: Ubuntu
Description:    Ubuntu 22.04.5 LTS
Release:        22.04
Codename:       jammy
```

We need it to run there.  It uses standard bash, nothing fancy.

Reply:
Ohh I actually made it sync with both windows and linux. I am accidentally using windows for sometime.

Let me know if you prefer me to revert that.,